string color
---
uint16 dumped
